### DISCUSSION TITLE
9.1.5 Source code

### DISCUSSION THREAD

**Discussion by iYawnz** [simulationcraft/simc#6654] [General]
Hey question, does someone have hands on the 9.1.5 Source code? Preferably latest build but any build is also fine. I could not find it in the release list which goes up to 8.3

thanks in advance!

**Comment by scamille** [comment]
We do no longer have any tag releases (only continous nightly builds). To find a specific version you'd have to check the file where the version number is defined in simc, which since quite a while is here: https://github.com/simulationcraft/simc/commits/shadowlands/engine/config.hpp

https://github.com/simulationcraft/simc/commit/d5793e2013ed73264d2e4dc72adaa31a2e5ba497 changed the version to 920, so maybe its parent or some other commit shortly before that would be what you are looking for.
